client/src/components/EmailModal/EmailModal.component.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Box,
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalBody,
+  ModalFooter,
+  ModalProps,
+  HStack,
+  Input,
+  Text,
+  VStack,
+  Textarea,
+} from '@chakra-ui/react'
+import { useState } from 'react'
+import { SubmitHandler, useForm } from 'react-hook-form'
+import { Agency } from '../../services/AgencyService'
+import { BiErrorCircle } from 'react-icons/bi'
+import { Enquiry } from '../CitizenRequest/CitizenRequest.component'
+
+interface EmailModalProps extends Pick<ModalProps, 'isOpen' | 'onClose'> {
+  onConfirm: (data: Enquiry) => Promise<void>
+  agency: Agency
+}
+
+export const EmailModal = ({
+  isOpen,
+  onClose,
+  onConfirm,
+  agency,
+}: EmailModalProps): JSX.Element => {
+  const { register, handleSubmit, reset, formState } = useForm()
+  const { errors: formErrors } = formState
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  const onSubmit: SubmitHandler<Enquiry> = async (data) => {
+    setIsLoading(true)
+    await onConfirm(data)
+    setIsLoading(false)
+    reset()
+    onClose()
+  }
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <ModalOverlay />
+        <ModalContent w="660px">
+          <ModalHeader>
+            <Text textStyle="h2" color="secondary.700">
+              {`${agency.shortname.toUpperCase()} Enquiry Form`}
+            </Text>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <VStack align="left" spacing={0}>
+              <Text>
+                {`This enquiry form will be sent to ${agency.longname}. Please do
+                note that we would take at least 3 working days to process your
+                enquiry. Thank you.`}
```

Comment:
Some indication here that this will generate an e-mail as an extra step would be helpful to the user

Reply:
@khaleedah can you look into how to reword this?